### ISSUE TITLE
How can I set up a readme-stats card with a special background？

### ISSUE THREAD

**Issue by toulzx** [moepoi/moepoi#2]
Hello~
I noticed that your readme-stats card has a nice background image. I'm really fond of it and eager to implement one myself. 
I have tried to do this by replacing the api interface, it seems that the “commits” data changed after the replacement. Perhaps because I used the parameters of the original api (*from [github-readme-stats](https://github.com/anuraghazra/github-readme-stats)*).
I don't have much experience in this area, would you please give me some points?

*btw. English is not my native language, so please forgive any possible errors.*

**Comment by BD103** [comment]
From some research, moepoi uses a fork on Github Readme Stats [here](https://github.com/Nekonify/github-readme-stats). It adds custom backgrounds, like so:

```
![Moepoi's github stats](https://bad-apple-github-readme.vercel.app/api?show_bg=1&username=moepoi)
```

**Comment by toulzx** [comment]
> From some research, moepoi uses a fork on Github Readme Stats [here](https://github.com/Nekonify/github-readme-stats). It adds custom backgrounds, like so:
> 
> ```
> ![Moepoi's github stats](https://bad-apple-github-readme.vercel.app/api?show_bg=1&username=moepoi)
> ```




**Thanks for your reply.**

I found that the datas displayed are somewhat different. And it makes me feel curious :(

_different `stars` are showed when with different `addr`:_

![Moepoi's github stats](https://github-readme-stats.vercel.app/api?show_bg=1&username=moepoi)

![Moepoi's github stats](https://bad-apple-github-readme.vercel.app/api?show_bg=1&username=moepoi)


_different `commits` when add api `include_all_commits`:_

![Tou's github stats](https://github-readme-stats.vercel.app/api?show_bg=1&username=toulzx&include_all_commits=true)

![Tou's github stats](https://bad-apple-github-readme.vercel.app/api?show_bg=1&username=toulzx&include_all_commits=true)


Apparently the `bad-apple-github-readme` was somewhat miscalculated.

**Comment by Arsybai** [comment]
@toulzx pppppp

**Comment by BD103** [comment]
@toulzx Yes, the fork is 201 commits behind. There must have been some calculation upgrades in the process. This repo hasn't been maintained since 2020.

**Comment by toulzx** [comment]
> @toulzx Yes, the fork is 201 commits behind. There must have been some calculation upgrades in the process. This repo hasn't been maintained since 2020.

WOW, I figured it out now, thank you so much for your hint! good luck!
